### ISSUE TITLE
Creating an organization for all charts

### ISSUE THREAD

**Issue by pmairoldi** [danielgindi/ChartsRealm#1]
Shouldn't we just create an organization for charts for ios, charts for android, and anything else related?

**Comment by danielgindi** [comment]
That's a great idea! @petester42 are you on board?

**Comment by pmairoldi** [comment]
Sure

**Comment by danielgindi** [comment]
Yeah I mean to tag @PhilJay :-)

**Comment by danielgindi** [comment]
I'm so busy today that I don't see what I read OR what I write

**Comment by liuxuan30** [comment]
lol I am baby sitting and typing on my phone in midnight.good idea

**Comment by Huang-Libo** [comment]
It's a great organization;-)

**Comment by PhilJay** [comment]
Sure thing, sounds like a good idea :-)

**Comment by PhilJay** [comment]
I guess it does not really matter who sets up the Organization but we should agree on a name

**Comment by danielgindi** [comment]
Yeah. Suggestions?

**Comment by PhilJay** [comment]
Well, "Charts" would probably be generic enough to describe everything and yet short and simple.
Not sure if it's an issue since the iOS part is already named that way.

**Comment by danielgindi** [comment]
Then the repos could be Charts/Swift, Charts/Android, Charts/SwiftRealm, Charts/AndroidRealm...

**Comment by PhilJay** [comment]
That could work, yes. But I'm not sure it's smart to rename the repositories concerning visibility?

**Comment by danielgindi** [comment]
Well whatever name we give them - we don't really need to change Cocoapod or Gradle names

**Comment by pmairoldi** [comment]
I think keeping the libraries names Charts is fine. We don't need to drop it and it aligns more with the actual name people will use in their code. As far as renaming goes, I'm pretty sure github redirects traffic to the new name automatically for some time.

**Comment by PhilJay** [comment]
Alright, sounds like we have a plan. Do you want me to create the organization?

**Comment by danielgindi** [comment]
Are we going to name it "Charts"?

**Comment by PhilJay** [comment]
That name works for me!

**Comment by pmairoldi** [comment]
Works for me too

**Comment by liuxuan30** [comment]
fine for me too

**Comment by danielgindi** [comment]
"Username is already taken". Next suggestion?

**Comment by Huang-Libo** [comment]
@danielgindi 
Maybe you should create a Team in Organization for better authority control, so you need to set a name for the Team. 🙃

**Comment by Huang-Libo** [comment]
@danielgindi 
Supplement: Not one team, maybe two team, 
- Android Charts Team 
 (contain Android Charts project and Android ChartsRealm project),
- iOS Charts Team 
 (contain iOS Charts project and iOS ChartsRealm project).

_And maybe there will has a **Web Charts Team** someday._

**Comment by pmairoldi** [comment]
Maybe use Mobile Charts instead. It's free. Or contact GitHub to get the username. It seems to belong to an inactive user.

**Comment by danielgindi** [comment]
> 
> Hi there Daniel,
> 
> Thanks for getting in touch with us. I've looked over the charts account and I'm afraid it's not dormant (not all activity on GitHub is public) or available for release under our name-squatting policy.
> 
> Sorry for the disappointment, please do let us know if you have any questions about this or anything else!
> 
> Thanks,
> Steve

**Comment by danielgindi** [comment]
I don't think that "MobileCharts" is entirely appropriate as the Swift version is available for macOS too ;)

**Comment by pmairoldi** [comment]
Ya I agree. Maybe like an adjective and then charts like "BeautifulCharts" or just like "ChartsOrg"

**Comment by PhilJay** [comment]
Hmmm, or something to highlight the cross-platform features?

**Comment by danielgindi** [comment]
MultiCharts? Even ChartsOrg is good enough as the "org" emphasizes it's not just one vertical

**Comment by pmairoldi** [comment]
ChartsOrg is fine by me
